ksqldb-engine/src/main/java/io/confluent/ksql/schema/registry/SchemaRegistryUtil.java

```diff
@@ -115,7 +115,7 @@ public static Optional<Integer> getLatestSchemaId(
     return getLatestSchema(srClient, subject).map(SchemaMetadata::getId);
   }
 
-  public static Optional<ParsedSchema> getLatestParsedSchema(
+  public static Optional<Integer> getSchemaId(
```

Comment:
This method is the same as the `getLatestSchemaId` above. What if you use the `getLatestSchemaId` instead?